sched/signal/sig_dispatch.c

```diff
@@ -319,7 +320,7 @@ int nxsig_tcbdispatch(FAR struct tcb_s *stcb, siginfo_t *info)
 
   /************************** MASKED SIGNAL ACTIONS *************************/
 
-  masked = (bool)sigismember(&stcb->sigprocmask, info->si_signo);
+  masked = (bool)nxsig_ismember(&stcb->sigprocmask, info->si_signo);
```

Comment:
Does this conversion to bool make sense? I think it does not account for error return values.

Reply:
In this case (and the following on), the signal number should be okay.  Probably a DEBUGASSERT(GOOD_SIGNO( info->si_signo)) is all we need.

Reply:
I changed it to make the cast correct.